lib/node_modules/@stdlib/_tools/doctest/compare-values/lib/main.js

```diff
@@ -435,6 +435,9 @@ function checkArray( actual, expected ) {
 					return false;
 				}
 			} else if ( isPrimitive( a ) ) {
+				if ( isArray( b ) ) {
```

Comment:
```suggestion
				if ( !isPrimitive( b ) ) {
```

Rather than test for an array here (which seems to me to be `<ndarray>` specific), shouldn't we be testing that `b` is also a primitive?

Reply:
@Orthodox-64 I would also recommend adding tests to verify that this change works as intended.